SourceX/DiabloUI/fonts.cpp

```diff
@@ -1,5 +1,6 @@
 #include "DiabloUI/fonts.h"
 #include "file_util.h"
+#include "../SourceS/paths.cpp"

```

Comment:
Don't include `cpp` files
I guess you're including it for the `TTF_FONT_NAME` define.

Instead, expose `TTF_FONT_NAME` as `GetTtfFontName()` and do the concatenation here.